src/tests/configur.sh

```diff
@@ -0,0 +1,219 @@
+#!/bin/sh
+
+#
+# Contributed by Ozkan Sezer <sezeroz@users.sourceforge.net>
+# for cross-compiling Watt-32 on Linux. Targets are:
+#   djgpp, mingw32, mingw64, cygwin, clang or watcom
+#
+# What works:
+# - generates suitable target makefile from makefile.all
+# - generates the src/build/<target> objects directory
+#
+# What does not work:
+# - generation of inc/sys/<target>.err, e.g. djgpp.err, and
+#   src/build/<target>/syserr.c won't happen:
+#   util/errnos.c relies on being compiled as a target-exe:
+#   it relies on the sys_nerr value from the target libc and
+#   the strerror() string returned from target's libc.  none
+#   of these can be accomplished by a simple cross-build.
```

Comment:
What does these comments have to do with `src/tests` files?

Reply:
I used configur.sh from src subdirectory and don't cleanup it. All compilers support except Watcom should be removed, because it is used only by watcom.